are.js.org

zosfong opened the pull request:
- [x] There is reasonable content on the page (see: [are-ruddy.vercel.app](https://are-ruddy.vercel.app/))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

MattIPv4 commented:
cc #7865

MattIPv4 commented:
Your last PR was closed due to lack of content, and looking at your site now, it still seems like most of the pages are just placeholders with TODO comments etc.?

zpfz commented:
> Your last PR was closed due to lack of content, and looking at your site now, it still seems like most of the pages are just placeholders with TODO comments etc.?

Site content is slowly updated with these days, or I write the content completed that  opening the issue.

MattIPv4 commented:
@indus do you think there is enough content here now?

indus commented:
@MattIPv4 Yes - I think that is fine.

@zosfong I've already added your requested subdomain to JS.ORGs zonefile. You may have to configure your host to accept the custom domain as I'm getting a 404 not found at the moment.
I'll merge after we have "contact". Please keep this PR open.

zosfong commented:
> @MattIPv4 Yes - I think that is fine.
> 
> @zosfong I've already added your requested subdomain to JS.ORGs zonefile. You may have to configure your host to accept the custom domain as I'm getting a 404 not found at the moment. I'll merge after we have "contact". Please keep this PR open.

i had added the domain in Vercel, it can preview correctly now.Thx a lot~

indus commented:
Welcome https://are.js.org
